src/scripts/main.js

```diff
@@ -18,10 +18,29 @@ const food = {
   },
 };
 
+const newTree = document.createElement('div');
```

Comment:
Consider adding a class or additional attributes to the `newTree` element for easier styling and manipulation later on.